### PR TITLE
Compatible with Crystal 0.25.0

### PR DESCRIPTION
**Pull request by icyleaf**
Please reviews, it passed all specs.

### CODE REVIEW

File: src/etcd/version.cr [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Etcd
-  VERSION = "0.2.0"
+  VERSION = "0.2.1"
```
Comment (teodor-pripoae) [inline src/etcd/version.cr:2]: Why not 0.2.0 ?

**Review by teodor-pripoae — CHANGES_REQUESTED** [review]

**Comment by icyleaf** [comment]
It had been released for a long time ago. I think v0.2.1 is good choices.

icyleaf
2018年7月30日 +0800 20:40 Teodor Pripoae <notifications@github.com>，写道：

**Comment by teodor-pripoae** [comment]
The current master version is 0.1.0.  You bumped it to 0.2.0 in your first commit.

**Comment by icyleaf** [comment]
Do you forget this version you released? https://github.com/kuende/etcd-crystal/releases/tag/v0.2.0

icyleaf
2018年7月30日 +0800 21:20 Teodor Pripoae <notifications@github.com>，写道：

**Comment by teodor-pripoae** [comment]
Correct. It seems I forgot to update shard.yml

**Pull request merged**